MapleServer2/Servers/Game/FieldManager.cs

```diff
@@ -239,12 +243,14 @@ public void AddPlayer(GameSession sender, IFieldObject<Player> player)
             }
 
             State.AddPlayer(player);
-
-            if (!State.HealingSpots.IsEmpty)
+            /*
+            replace if(true) with a good/relevant condition to initiate the mapUpdatesTask
+            */
+            if (true)
             {
-                if (HealingSpotThread == null)
+                if (MapLoopTask == null)
```

Comment:
Since it is within AddPlayer, it will not be null after the first player joined.
MapLoopTask should be at a better place but for now AddPlayer seems like the least bad fit for me.